opm/input/eclipse/Schedule/Group/Group.cpp

```diff
@@ -34,18 +34,18 @@
 #include <utility>
 
 namespace {
-    bool has_limit(const double x)
+    bool is_defined(const double x)
     {
-        return std::abs(x) > 0.0;
+        return  x != Opm::RestartIO::RstGroup::UNDEFINED_VALUE;
```

Comment:
I'm a little uneasy about this kind of comparison.  `x` is type `double` whereas `UNDEFINED_VALUE` is type `float` and will therefore be promoted to a `double` for `!=`.  Maybe a slightly more relaxed condition of the form `x < UNDEFINED_VALUE / 2` could be appropriate here to ensure we don't have false positives?

Reply:
Good point, updated now.